### PR TITLE
fix: added properties file for easier configuration

### PR DESCRIPTION
**Pull request by FeKozma**
### Summary

- \# 1 [Added properies file for easier configuration](https://github.com/FeKozma/Code-check/pull/9/commits/b2c3770fe22d26dee8d31a3bd67922dd2a271cbd)
- \# 2 [Fixed LOGGING_LEVEL priority](https://github.com/FeKozma/Code-check/pull/9/commits/d12ca07aa97d59ddbd08658fa0bec435acdd46d5)
- \# 3 [Added RESULT_NAME_PREFIX so you can change the prefix of result files](https://github.com/FeKozma/Code-check/pull/9/commits/bf88b77ef1fe969772426d24c7bfee53a76639a1)
- \# 4 [Fixed some results not showing up in CheckDirectory.java](https://github.com/FeKozma/Code-check/pull/9/commits/ea089e0f85934f598376aa40a701537e2d1bfe30)
- \# 5 [Added info for how long LLM has running for when it's finished](https://github.com/FeKozma/Code-check/pull/9/commits/f26530666d084451061db417ff4ba322a00bdc7d)
- \# 6 [Renamed the Util interface to Log, since it's used for logging now](https://github.com/FeKozma/Code-check/pull/9/commits/f4b5c9c2ea9afb78dbec5951479c6257a499bcaa)
- \# 7 [Added EXCLUDED_PATHS to configuration](https://github.com/FeKozma/Code-check/pull/9/commits/e2b914f4468923a4497c238b1df93f7cd8e83dab)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,11 +1,13 @@
 .gradle
 build/
+local-config.properties
+model/
 !gradle/wrapper/gradle-wrapper.jar
 !**/src/main/**/build/
 !**/src/test/**/build/
 
 ### IntelliJ IDEA ###
-.idea/*
```
Comment (Livila) [inline .gitignore:8]: Maybe `.idea/` to show that it's a folder.

Comment (FeKozma) [reply .gitignore:8]: I dont understand this line, i thought * would take anything in folder

**Comment by Livila** [comment]
`LOGGING_LEVEL` is broken.

File: src/main/java/CodeCheck/ConfigInterface.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+            NONE (0),
+            ERROR(1),
+            ERR(1),
+            E(1),
+            WARNING(2),
+            WARN(2),
+            W(2),
+            DEBUG(3),
+            D(3),
+            INFO(4),
```
Comment (Livila) [inline src/main/java/CodeCheck/ConfigInterface.java:21]: Switch `INFO` with `DEBUG`.
`INFO` -> 3
`DEBUG` -> 4

File: .gitignore (hunk 1 above)
Comment (Livila) [reply .gitignore:8]: Yeah, just keep it as is. `.grade` above has the same as `.idea` now anyways.

File: src/main/java/CodeCheck/ConfigInterface.java [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+            I(4),
+            TRACE(5),
+            T(5);
+
+            private int lvl;
```
[the remark this answers is not in the document]
Comment (Livila) [reply src/main/java/CodeCheck/ConfigInterface.java]: Done.

File: src/main/java/CodeCheck/ConfigInterface.java [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+import java.io.File;
+import java.io.FileInputStream;
+import java.util.Properties;
+
+public interface ConfigInterface {
+    Config conf = new Config();
+
+    class Config {
+        enum LoggingLevel {
+            NONE (0),
```
[the remark this answers is not in the document]
Comment (Livila) [reply src/main/java/CodeCheck/ConfigInterface.java]: Done.

**Review by Livila — APPROVED** [review]
Looks good.

File: README.md
Comment (FeKozma) [inline README.md:1]: nice fixed 😅

**Pull request closed without merging**